### ISSUE TITLE
Integrated Tunnels Exporters Impossible to Craft

### ISSUE THREAD

**Issue by Ioperf** [FTBTeam/FTB-Interactions#674]
## Modpack Version: 
1.7.0

## Bug Report:

Any of the Integrated Tunnels importers(except fluid and item) cannot be changed into exporters in a crafting table. I have tried every slot in the crafting bench and even using a freshly crafted importer. Instead of an exporter you get another importer.

## Expected Behavior/Suggestions:

According to JEI you should be able to put the importer in any slot in a crafting bench and turn it into the exporter.

Other than making this recipe function, if that does not work add an alternate recipe perhaps in the assembler or and alternate crafting recipe with an extra item like a wrench or dye or something.
## Is It Repeatable? Steps to Reproduce: 

1. Take any kind of importer from Integrated Tunnels
2. Place into any slot in a crafting bench
3. Receive a brand new importer of the same kind

## Mod/s Affected: 

Integrated Tunnels

**Comment by TomyLobo** [comment]
I also have that problem.
A solution might be a positional recipe, like the recipes for tiny/small pile of dust.

**Comment by desagas** [comment]
Issue was / is not fixed in 1.9.1, which was released after this was closed.

**Comment by dronf** [comment]
None of the cross item ID recipes seem to be working.  Importer makes importer, exporter makes exporter, no matter what position in the grid is used.

**Comment by desagas** [comment]
I know a lot of the FTb Team live in parts of the world hotter than hot right now, so I’m assuming they have not seen the replies to this threat. Hopefully they open it again and add like a assembler recipe that costs almost nothing or something like that.

Denver Leman

On Aug 8, 2019, at 8:40 PM, dronf <notifications@github.com<mailto:notifications@github.com>> wrote:


None of the cross item ID recipes seem to be working. Importer makes importer, exporter makes exporter, no matter what position in the grid is used.

**Comment by TomyLobo** [comment]
or just a positional recipe, like for the small/tiny piles

**Comment by dronf** [comment]
> or just a positional recipe, like for the small/tiny piles

It doesn't work in any slot, on any type of crafting table.  If that worked people wouldn't still be complaining about it 6 weeks after it was claimed to be fixed.

**Comment by TomyLobo** [comment]
@dronf I was suggesting the devs add a recipe. I edited my post for clarity

**Comment by desagas** [comment]
There is a recipe that has been added for shapeless crafting. That is why it shows up in JEI, however, it interferes with the mods own recipe that is used to clear not data from them, which takes priority. That is why a simple shapeless recipe will not work.

Denver Leman

On Aug 10, 2019, at 12:22 AM, TomyLobo <notifications@github.com<mailto:notifications@github.com>> wrote:


@dronf<https://github.com/dronf> I was suggesting the devs add a recipe. I edited my post for clarity
